lms/djangoapps/support/views/course_entitlements.py

```diff
@@ -6,20 +6,43 @@
 from django.db.models import Q
 from django.http import HttpResponseBadRequest
 from django.utils.decorators import method_decorator
+from django.views.generic import View
 from edx_rest_framework_extensions.authentication import JwtAuthentication
 from rest_framework import permissions, status, viewsets
 from rest_framework.response import Response
 
+from edxmako.shortcuts import render_to_response
 from entitlements.api.v1.permissions import IsAdminOrAuthenticatedReadOnly
 from entitlements.api.v1.serializers import SupportCourseEntitlementSerializer
 from entitlements.models import CourseEntitlement, CourseEntitlementSupportDetail
+from lms.djangoapps.commerce.utils import EcommerceService
 from lms.djangoapps.support.decorators import require_support_permission
 from openedx.core.djangoapps.cors_csrf.authentication import SessionAuthenticationCrossDomainCsrf
 
 REQUIRED_CREATION_FIELDS = ['course_uuid', 'reason', 'mode']
 
 
-class EntitlementSupportView(viewsets.ModelViewSet):
+class EntitlementSupportView(View):
+    """
+    View for viewing and changing learner enrollments, used by the
+    support team.
+    """
+    @method_decorator(require_support_permission)
+    def get(self, request):
+        """Render the enrollment support tool view."""
+        support_actions = CourseEntitlementSupportDetail.get_support_actions_list()
+
+        ecommerce_url = EcommerceService().get_order_dashboard_url()
+        context = {
+            'username': request.GET.get('user', ''),
+            'uses_bootstrap': True,
```

Comment:
It doesn't look like this or `username` are getting used as `prop`s in the `renderReact` method - are they getting pulled from the `context` for some other purpose?

Reply:
RIght, @McKenzieW we should chat about this. If I included the username here (which in this context is the support user), I could include 'support user' in post and patch requests that create support details, which I may recall being easier/cleaner on the back end? otherwise I will remove the username from the context.

Reply:
actually @jaebradley its probably used for the header, 
<img width="1076" alt="screen shot 2018-03-09 at 2 06 03 pm" src="https://user-images.githubusercontent.com/15352505/37224992-2b7f4eec-23a3-11e8-969b-1ee05d1dff9a.png">